test/extended/authorization/podsecurity_admission.go

```diff
@@ -189,14 +193,302 @@ var _ = g.Describe("[sig-auth][Feature:PodSecurity][Feature:SCC]", func() {
 	})
 })
 
+var _ = g.Describe("[sig-auth][Feature:PodSecurity][Feature:LabelSyncer][Feature:ForceHistoricalOwnership]", func() {
+	defer g.GinkgoRecover()
+
+	ctx := context.Background()
+	oc := exutil.NewCLIWithoutNamespace("pod-security-managed-labels-historical")
+
+	cpc := "cluster-policy-controller"
+	psaLabelSyncer := "pod-security-admission-label-synchronization-controller"
+	ocManager := "openshift-tests"
```

Comment:
don't rely on things you don't control

Reply:
@stlaz I couldn't find a way to retrieve this value dynamically so I hard-coded it; is there a way?

The value of `ocManager` is only used to verify the expected manager after an update to the labels; this is the manager when the `oc := exutil.NewCLIWithoutNamespace()` client is used to perform a `Create()` or `Update().

Reply:
I've changed this to use an explicit manager when updating as well (not only for create), in 43c3fbc5c65472f0cebbec9b7a285c4f5725db20